src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -408,7 +416,7 @@ private void GetLocationByGraphNode(ref int line, ref int col)
                 var argsJoined = string.Join(", ", arguments.Select(GetTypeName));
                 
                 var fep = funcGroup.FunctionEndPoints[0];
-                var formalParamsJoined = string.Join(", ", fep.FormalParams.Select(x => x.ToShortString()));
+                var formalParamsJoined = string.Join(", ", fep.FormalParams);
```

Comment:
is this going to use full class names when throwing function not found exceptions?

Reply:
Yes, this is the intended fix for https://jira.autodesk.com/browse/DYN-1044.